package.json

```diff
@@ -30,8 +30,8 @@
     "fh-wfm-user": "0.6.0",
     "fh-wfm-workflow": "0.2.2",
     "fh-wfm-workorder": "0.2.2",
-    "fh-wfm-sync": "0.1.2",
-    "raincatcher-mongoose-store": "0.1.0-pre.1",
+    "fh-wfm-sync": "0.2.0-pre.1",
+    "raincatcher-mongoose-store": "0.1.0-pre.2",
```

Comment:
Going to move package name to fh-wfm-mongoose-store to be consistent and remove previous one from npm.